### DISCUSSION TITLE
Mix of continuous and categorical variables association rules

### DISCUSSION THREAD

**Discussion by PARODBE** [rasbt/mlxtend#1007] [Q&A]
Hi,

Is there any option of use association rules when you have a mix of variables continuous and categorical? Like CBA algorithm... Thanks

**Comment by rasbt** [comment]
Good question, something like this is currently not implemented. For the current implementations, the items have to be discrete categories.

**Reply by PARODBE** [reply]
And do you think that It could be posible? Are there other libraries like CBA of arules in r or pyarc in python. For It IS of your interest

**Reply by NimaSarajpoor** [reply]
@PARODBE 
I think there are some packages in R (e.g. see [CBA](https://rdrr.io/cran/arulesCBA/man/CBA.html)). As explained in the package, one idea is to discretize the continuous variable to make it categorical. Then, you should(?) be able to use existing version of mlxtend for association rule discovery.

**Reply by PARODBE** [reply]
Thanks @NimaSarajpoor I know this option but sometimes when you discretize a continuous variable you could loss information, for this I prefer a methods which incorporates the option of working with continuous variables...

**Reply by rasbt** [reply]
Yes, discreticing the continuous variables as @NimaSarajpoor suggested would be the easiest. An option/algorithm that support continuous variables directly would of course be nice, but I currently wouldn't have the capacity to work on something like that in the foreseeable future, unfortunately.
